src/app/page.tsx

```diff
@@ -1,9 +1,44 @@
+'use client'
```

Comment:
let's keep the main page file simple.
Please move this functions in the @/src/app/components/home/Home.tsx component